lab_1/main.py

```diff
@@ -40,32 +73,73 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list or None:
     :param top_n: a number of the most common words
     :return: a list of the most common words
     """
-    pass
+    # Check for bad input
+    if (not isinstance(freq_dict, dict)
+            or not isinstance(top_n, int)):
+        return None
+    # Convert frequency dictionary to list of tuples,
+    # and sort list by frequency.
+    tokens = sorted(freq_dict.items(), key=lambda x: x[1], reverse=True)
```

Comment:
I'm just curious, why do you need items instead of keys?
As I can see here – `tokens = [word for word, freq in tokens]`, you remove values afterwards
Then why not sort keys instead of keys+values?

And then you can get top_n just here